src/pocketmine/entity/Villager.php

```diff
@@ -33,6 +33,25 @@ class Villager extends Creature implements NPC, Ageable{
 	public const PROFESSION_BLACKSMITH = 3;
 	public const PROFESSION_BUTCHER = 4;
 
+	/**
+	 * Can be used in @see $traderName
+	 * Automatically translated on the client side
+	 */
+	public const DEFAULT_NAME = "entity.villager.name";
+	public const ARMORER = "entity.villager.armor";
```

Comment:
shouldn't it be "entity.villager.armorer"?

Reply:
This is from game resources